[master < T2358-CL] Add docs for new GSS functions: Round, Floor, and Ceil (#474)
memgraph/docs: lab/style-script/gss-functions.md

- `number`

### `Floor(value)`

> Returns the largest integer not greater than the input value.

Reviewer: ```suggestion
Returns the largest integer less than or equal to the input value.
```

Reviewer: For me, it is more clear without negation. But, if you prefer _not greater_, you can leave it like that.

Author: Definitely agree, it is more clear without negation!
